setup.py

```diff
@@ -33,7 +33,11 @@
         'jinja2 >=2.10, <3.0',
         'graphviz >=0.13, <1.0',
         'pyflakes >=2.1.1',
-        'dill >=0.3, <0.4'
+        'dill >=0.3, <0.4',
+        'IPython >= 7.6.0',
+        'jupyter-client >= 5.3.4',
```

Comment:
+ jupyter-core >= 4.6.0
+ kfp-server-api == 0.1.18.3

Reply:
We can also go for
+ kfp >= 0.2

Reply:
+ `packaging` is a pytest requirement, which we only install on `dev` installation, but you're using it on `jupyter_utils`, so you should also add an entry for that here.